vic/drivers/image/src/get_global_param.c

```diff
@@ -838,6 +841,9 @@ get_global_param(FILE *gp)
             else if (strcasecmp("N_OUTFILES", optstr) == 0) {
                 ; // do nothing
             }
+            else if (strcasecmp("COORD_DIMS_OUT", optstr) == 0) {
+                ; // do nothing
+            }
```

Comment:
You'll never get to this `else if` because of the lines 780-782.  Which do you want?